api/swagger.yml

```diff
@@ -487,10 +536,16 @@ paths:
           required: false
           schema:
             $ref: '#/components/schemas/BJJ'
-        - name: accountIndex
+        - name: fromAccountIndex
```

Comment:
accountIndex filter should be keeped, otherwise we're breaking the API contract, and will force us to move to `v2`